components/sections/homepage/Gallery.jsx

```diff
@@ -0,0 +1,175 @@
+/* eslint-disable @next/next/no-img-element */
+import { useRef, useState, useEffect } from "react";
+import Wrapper from "../../layout/Wrapper";
+import { Stage, Layer, Image, Rect } from "react-konva";
+
+export default function Gallery() {
+  let [coods, setCoods] = useState([]);
+  let images = [useRef(), useRef(), useRef()];
```

Comment:
Is not the best practice to put the state into an array as `[useRef(), useRef(), useRef()]`, it will be good if you create `imagesArray` the have one `useRef`. this link [here](https://stackoverflow.com/questions/54633690/how-can-i-use-multiple-refs-for-an-array-of-elements-with-hooks) can be helpfull.